Dockerfile

```
@@ -6,9 +6,10 @@ ENV PYTHONUNBUFFERED 1
 
 WORKDIR /code
 
-COPY ./requirements.txt /code/requirements.txt
+COPY ./Pipfile* /code
```

Let's use this, wildcard is risky.
```suggestion
COPY Pipfile Pipfile.lock /code/
```